src/main/python/rlbot/utils/process_configuration.py

```diff
@@ -71,3 +71,18 @@ def extract_all_pids(agent_metadata_map):
     for player_index, data in agent_metadata_map.items():
         pids.update(data.pids)
     return pids
+
+
+def is_process_running(program, scriptname):
+    if not optional_packages_installed:
+        return True
+    for pid in psutil.pids():
```

Comment:
Consider using `psutil.process_iter` instead, it would be less memory heavy and a bit faster as you would not need to call `psutil.Process` on every pid